ipywidgets/widgets/widget_selection.py

```diff
@@ -304,7 +304,7 @@ def __init__(self, *args, **kwargs):
         # so we don't have errors the first time validators run
         self._initializing_traits_ = True
         options = _make_options(kwargs.get('options', ()))
-        self._full_options = options
+        self._options_full = options
```

Comment:
Good catch! Can you also add the _make_options call, like when it is assigned other places?
```suggestion
        self._options_full = _make_options(options)
```

Reply:
Oh, never mind, `_make_options` is called in the line above.